tests/unit/store/asserts-test.js

```diff
@@ -0,0 +1,36 @@
+import {createStore} from 'dummy/tests/helpers/store';
+
+import {module, test} from 'qunit';
+
+module("unit/store/asserts - DS.Store methods produce useful assertion messages");
+
+const MODEL_NAME_METHODS = [
+  'createRecord',
+  'findRecord',
+  'findByIds',
+  'peekRecord',
+  'hasRecordForId',
+  'recordForId',
+  'query',
+  'queryRecord',
+  'findAll',
+  'peekAll',
+  'filter',
+  'recordIsLoaded',
+  'modelFor',
+  'modelFactoryFor',
```

Comment:
`recordIsLoaded` is marked as private in the API docs; `modelFactoryFor` currently has no public documentation (though the name suggests it is public).

Tiny nitpick: I wonder if those methods should be excluded from the assertion tests here, since they are not public API?